docs/guides/set-up-webhooks.md

```diff
@@ -0,0 +1,153 @@
+# Webhooks for flexible scheduling notifications
+Webhooks are the easiest and fastest solution for quick notification. If something happens, they send the automated message to particular URL. It works for notification and also integration with different platforms. So, how does it work in Vyte?
```

Comment:
"an automated to a particular"

Reply:
"It is useful for notifications and integrating with other platforms"

Reply:
done 04ac6d4c68efa1e05553386bbc2d8e0c28248c37